scripts/build.py

```diff
@@ -1325,7 +1325,7 @@ def compile_typescript_files_continuously(project_dir):
     safe_delete_directory_tree(COMPILED_JS_DIR)
     python_utils.PRINT('Compiling ts files in watch mode...')
     cmd = [
-        './node_modules/typescript/bin/tsc', '--project', project_dir,
+        'node', './node_modules/typescript/bin/tsc', '--project', project_dir,
```

Comment:
Hmm, what happens if they don't install node on their systems? It might be best to reference our version of node.

Reply:
No, this will execute the `node` in `oppia_tools`, since we have updated the path in the `common.py` script. 

```
In [1]: import subprocess

In [2]: subprocess.Popen(['which', 'node'])
/usr/bin/node
Out[2]: <subprocess.Popen at 0x7f3fc6f79b10>

In [3]: subprocess.Popen(['node', '--version'])
Out[3]: <subprocess.Popen at 0x7f3fc6f8c190>

v13.2.0
In [4]: import scripts.common

In [5]: subprocess.Popen(['which', 'node'])
/home/cuichen/opensource/oppia/../oppia_tools/node-10.15.3/bin/node
Out[5]: <subprocess.Popen at 0x7f3fc63ab7d0>

In [6]: subprocess.Popen(['node', '--version'])
Out[6]: <subprocess.Popen at 0x7f3fc63ced10>

v10.15.3
```

PS: this reminds me that the things we have done on the `coverage` issues was non-necessary. I will add a comment in that PR.

Reply:
Understood -- but for safety, any issues with using the explicit node path in oppia_tools? I understand that we've modified the path but I feel like that's a bit fragile and it's easy for someone to change that in the future and not realize the knock-on effects on this code.

Reply:
Only one minor issue, the path to the node is different between windows and the other os. But I can add a condition check to tweak it.

Reply:
Updated